Jenkinsfile

```diff
@@ -488,6 +499,91 @@ gcloud container clusters create ${clusterName} \
+                                    def dnsPrefix = "${clusterName}"
+                                    def adminEmail = "${clusterName}@${parentZone}"
+                                    def dnsZone = "${dnsPrefix}.${parentZone}"
+
+                                    try {
+                                        runIntegrationTest(platform, "eks --config=${clusterName}-autogen.json --dns-zone=${dnsZone} --email=${adminEmail}", "--dns-suffix ${dnsZone}") {
+                                            ansiColor('xterm') {
+                                                sh """
+eksctl create cluster \
+ --name ${clusterName} \
+ --region ${region} \
+ --version ${kversion} \
+ --node-type m5.large \
+ --nodes-min 3 \
+ --timeout 60m \
+ --kubeconfig \$KUBECONFIG \
+ --tags 'platform=${platform},branch=${BRANCH_NAME},build=${BUILD_URL}'
+"""
+
+                                                waitForRollout("kube-system", 30)
+                                            }
+
+                                            // NB: writeJSON doesn't work without approvals(?)
+                                            // See https://issues.jenkins-ci.org/browse/JENKINS-44587
+                                            writeFile([file: "${clusterName}-autogen.json", text: """
+{
+  "dnsZone": "${dnsZone}",
+  "contactEmail": "${adminEmail}",
+  "externalDns": {
+    "aws_access_key_id": "${AWS_ACCESS_KEY_ID}",
+    "aws_secret_access_key": "${AWS_SECRET_ACCESS_KEY}"
+  },
+  "oauthProxy": {
+    "client_id": "${fixme}",
+    "client_secret": "${fixme}",
+    "cookie_secret": "${fixme}",
+    "aws_region": "${region}",
+    "aws_user_pool_id": "${region}_${fixme}"
+  }
+}
+"""
+                                            ])
+
+                                            writeFile([file: 'kubeprod-manifest.jsonnet', text: """
+(import "manifests/platforms/eks.jsonnet") {
+  config:: import "${clusterName}-autogen.json",
+  letsencrypt_environment: "staging",
+  prometheus+: import "tests/testdata/prometheus-crashloop-alerts.jsonnet",
+}
+"""
+                                            ])
+                                        }
+                                    }{
+                                        // update glue records in parent zone
+                                        container('aws') {
+                                            def output = sh(returnStdout: true, script: """
+id=$(aws route53 list-hosted-zones --query 'HostedZones[?Name==`bkpr.run.`].Id' --output text)
+aws route53 get-hosted-zone --id $id --query DelegationSet.NameServers
+""")
+                                            def nameServers = readJson(text: output)
+                                            insertGlueRecords(dnsPrefix, nameServers, "60", parentZone, parentZoneResourceGroup)
+                                        }
+                                    }
+                                }
```

Comment:
oops, looks like I forgot to add cleanup code in a `finally` unwind about here.  We should call `eksctl delete cluster ...`

Reply:
Done.

Reply:
Fixed.